### DISCUSSION TITLE
Mask is not being transformed (only image)

### DISCUSSION THREAD

**Discussion by Geethen** [kornia/kornia#2956] [Potential Issue] [answered]
I am trying to apply the [flipnslide](https://github.com/elliesch/flipnslide) augmentation, however, the image only get transformed. I have tried passing the inputs as a dict and as separate args but the output mask remains the same shape as the input. Instead I am expecting the output tensor to have a shape of (58, 1, 64,64). See reproducible example below.
```
import torch
from flipnslide.tiling import FlipnSlide
from kornia.augmentation.container import AugmentationSequential
from kornia.constants import DataKey


class FlipnSlideAug(torch.nn.Module):
    def __init__(self, tilesize: int, viz: bool = False):
        super().__init__()
        self.tilesize = tilesize
        self.viz = viz
    
    def forward(self, tensor):
        # Process the input tensor directly
        return self.process_tensor(tensor)
    
    def process_tensor(self, tensor):
        # Ensure input is on CPU and convert to numpy
        np_array = tensor.squeeze(0).cpu().numpy()
        
        sample_tiled = FlipnSlide(
            tile_size=self.tilesize, 
            data_type='tensor',
            save=False, 
            image=np_array,
            viz=self.viz
        )
        
        # Return the tiles directly
        return sample_tiled.tiles

# Usage with AugmentationSequential
flipnslide = FlipnSlideAug(tilesize=64, viz=False)
tfms = AugmentationSequential(
    flipnslide,
    data_keys=[DataKey.INPUT, DataKey.MASK]# Specify both keys
)

# Example usage
train_batch = {
    'image': torch.rand(1, 3, 256, 256),
    'mask': torch.rand(1, 1, 256, 256)
}

transformed = tfms(train_batch['image'], train_batch['mask'])

print("Transformed image shape:", transformed[0].shape)
print("Transformed mask shape:", transformed[1].shape)
```

OUTPUT
Transformed image shape: torch.Size([58, 3, 64, 64])
Transformed mask shape: torch.Size([1, 1, 256, 256])

**Comment by edgarriba** [comment]
@johnnv1  @shijianjian any idea why this could happen ?

**Reply by Geethen** [reply, marked as answer]
Problem fixed. I needed to subclass the geometricaugmentation instead: [torchgeo discussion](https://github.com/microsoft/torchgeo/discussions/2172)
